tests/functional/object/mcg/test_write_to_bucket.py

```diff
@@ -456,3 +459,43 @@ def test_content_encoding_with_write(
         logger.info(
             "Put object operation is preserving ContentEncoding as a object metadata"
         )
+
+    @tier2
+    @bugzilla("2259189")
+    @bugzilla("2264480")
+    @pytest.mark.polarion_id("OCS-5773")
+    def test_nb_db_activity_logs_on_io(
+        self, bucket_factory, awscli_pod_session, mcg_obj
+    ):
+
+        """
+        This test checks if the activity logs are being logged
```

Comment:
Could you please add more detail to the description. i.e., for default log level, we don't expect to log creation/deletion operations.